src/sdk/src/fragment-components/a-numberinput.tsx

```diff
@@ -147,6 +147,9 @@ export const ANumberInput = ({
 	componentObj,
 	...rest
 }: any) => {
+	const preventClickStyle = css`
+		pointer-events: none;
+	`;
```

Comment:
Move class outside of the function. Do the same for all of the files below as well

Reply:
done